pkgs/NetApp/trident/registry.yaml

```diff
@@ -0,0 +1,19 @@
+packages:
+  - type: github_release
+    repo_owner: NetApp
+    repo_name: trident
+    name: NetApp/trident/tridentctl
```

Comment:
Because the `trident/trident-installer/extras/bin` contains an executable binary named `trident`.
ref. https://github.com/aquaproj/aqua-registry/pull/7008#issuecomment-1276906124
```
$ ./trident --help
Usage of ./trident:
  -address string
        Storage orchestrator HTTP API address (default "127.0.0.1")
  -aes_key string
        AES encryption key (default "/certs/aesKey")
  -config string
        Path to configuration file(s)
  -crd_persistence
        Uses CRDs for persisting orchestrator state.
  -csi_endpoint string
        Register as a CSI storage provider with this endpoint
  -csi_node_name string
        CSI node name
  -csi_role string
        CSI role to play: 'controller' or 'node'
  -csi_unsafe_detach
        Prefer to detach successfully rather than safely
  -debug
        Enable debugging output
  -docker_plugin_mode
        Enable docker plugin mode
  -driver_port string
        Listen on this port instead of using a Unix domain socket
  -http_request_timeout duration
        Override the HTTP request timeout for Trident controller’s REST API (default 1m30s)
  -https_address string
        Storage orchestrator HTTPS API address
  -https_ca_cert string
        HTTPS CA certificate (default "/certs/caCert")
  -https_client_cert string
        HTTPS client certificate (default "/certs/clientCert")
  -https_client_key string
        HTTPS client private key (default "/certs/clientKey")
  -https_port string
        Storage orchestrator HTTPS API port (default "8443")
  -https_rest
        Enable HTTPS REST interface
  -https_server_cert string
        HTTPS server certificate (default "/certs/serverCert")
  -https_server_key string
        HTTPS server private key (default "/certs/serverKey")
  -k8s_api_server string
        Kubernetes API server address to enable dynamic storage provisioning for Kubernetes.
  -k8s_config_path string
        Path to KubeConfig file.
  -k8s_pod
        Enables dynamic storage provisioning for Kubernetes if running in a pod.
  -kubeconfig string
        Paths to a kubeconfig. Only required if out-of-cluster.
  -log_format string
        Logging format (text, json) (default "text")
  -log_level string
        Logging level (debug, info, warn, error, fatal) (default "info")
  -metrics
        Enable metrics interface
  -metrics_address string
        Storage orchestrator metrics address
  -metrics_port string
        Storage orchestrator metrics port (default "8001")
  -no_persistence
        Does not persist any metadata.  WILL LOSE TRACK OF VOLUMES ON REBOOT/CRASH.
  -node_prep
        Attempt to install required packages on nodes. (default true)
  -passthrough
        Uses the storage backends as the source of truth.  No data is stored anywhere else.
  -port string
        Storage orchestrator HTTP API port (default "8000")
  -rest
        Enable HTTP REST interface (default true)
  -volume_driver string
        Register as a Docker volume plugin with this driver name (default "netapp")
```